piq/gmsd.py

```diff
@@ -165,12 +265,14 @@ def __init__(self, reduction: str = 'mean', data_range: Union[int, float] = 1.,
 
         # Loss-specific parameters.
         self.data_range = data_range
-        
+
         # Values from the paper
         if scale_weights is None:
             self.scale_weights = torch.tensor([0.096, 0.596, 0.289, 0.019])
-        else:
+        elif isinstance(scale_weights, torch.Tensor):
             # Normalize scale weights
+            self.scale_weights = scale_weights / scale_weights.sum()
+        else:
```

Comment:
You already have this check inside `multi_scale_gmsd` function. Propose not duplicate it here. 
Just save scale_weights:
`self.scale_weights = scale_weights`